playbooks/bhyve.yaml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Manage FreeBSD bhyve VMs
+  hosts: megatron
+  gather_facts: no
+  become: yes
+
+  vars:
+    vm_list:
+      - name: k3s01
+        ram: 4G
+        cpus: 2
+        iso_path: /zstor/vm/.iso/k3os-amd64.iso
+      - name: k3s02
+        ram: 4G
+        cpus: 2
+        iso_path: /zstor/vm/.iso/k3os-amd64.iso
+      - name: k3s03
+        ram: 4G
+        cpus: 2
+        iso_path: /zstor/vm/.iso/k3os-amd64.iso
+
+  tasks:
+    - name: Create ZFS Datasets
+      zfs:
+        name: "{{ item.name }}"
+        state: present
+        mountpoint: "/zstor/vm/{{ item.name }}"
+      with_items: "{{ vm_list }}"
+
+    - name: Start VMs
+      command: >
+        bhyve -c {{ item.cpus }} -m {{ item.ram }} -A -H -P
+        -s 0:0,hostbridge -s 1:0,virtio-net,vm_if
```

Comment:
need to figure out the right bhyve syntax here, this is just copy-pasta'd from the example....

OR look at https://github.com/churchers/vm-bhyve or https://www.bsdstore.ru/en/about.html